src/PKSim.Infrastructure/Serialization/Xml/Serializers/ApplicationSettingsXmlSerializer.cs

```diff
@@ -14,6 +14,7 @@ public ApplicationSettingsXmlSerializer()
 
       public override void PerformMapping()
       {
+         Map(x => x.MoBiPath);
```

Comment:
User can now specify the Path of the MoBi executable in the app settings (will be used as fall back option if not found in registry)